src/AutoRest.CSharp/Common/Output/Models/Types/ModelTypeProvider.cs

```diff
@@ -112,31 +119,27 @@ private ConstructorSignature EnsurePublicConstructorSignature()
             if (_inputModel.DiscriminatorPropertyName is not null)
                 accessibility = MethodSignatureModifiers.Protected;
 
-            FormattableString[] baseInitializers = GetInitializersFromParameters(parametersToPassToBase);
-
             return new ConstructorSignature(
                 Type,
                 $"Initializes a new instance of {Type:C}",
                 null,
                 accessibility,
                 fullParameterList,
-                Initializer: new(true, baseInitializers));
```

Comment:
remove the invocation of `GetInitializersFromParameters` because the ctor of `Initializer` does exactly the same thing.